ui/app/templates/components/dashboard/vault-configuration-details-card.hbs

```diff
@@ -0,0 +1,38 @@
+<h3 class="title is-4" data-test-configuration-details-title>Configuration details</h3>
+
+<Hds::Table @caption="Vault configuration details" class="is-border-spacing-revert">
+  <:body as |B|>
+    <B.Tr>
+      <B.Td>API_ADDR</B.Td>
+      <B.Td data-test-vault-config-details="api_addr">{{@vaultConfiguration.api_addr}}</B.Td>
+    </B.Tr>
+    <B.Tr>
+      <B.Td>Default lease TTL</B.Td>
+      <B.Td data-test-vault-config-details="default_lease_ttl">{{format-duration
+          @vaultConfiguration.default_lease_ttl
+        }}</B.Td>
+    </B.Tr>
+    <B.Tr>
+      <B.Td>Max lease TTL</B.Td>
+      <B.Td data-test-vault-config-details="max_lease_ttl">{{format-duration @vaultConfiguration.max_lease_ttl}}</B.Td>
+    </B.Tr>
+    <B.Tr>
+      <B.Td>TLS</B.Td>
+      <B.Td
+        data-test-vault-config-details="tls_disable"
+      >{{@vaultConfiguration.listeners.firstObject.config.tls_disable}}</B.Td>
```

Comment:
Will this always be the first object?

Reply:
https://developer.hashicorp.com/vault/docs/configuration/listener/tcp

Reply:
Wondering if instead we should be searching for the element that has the `config` key 🤔

Reply:
I just pushed up my changes [here](https://github.com/hashicorp/vault/pull/21674/files#diff-920280532a140d32432addda632d130b99e4ce98eccd1663aa346c97f043cb99). For now to get the PR merged to the sidebranch, I think we can keep it where we search for the element that has the config key. I'll make a ticket to get backend input :)